modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -291,6 +291,11 @@ class Editor implements EditorObservable {
       DOMUtils.DOM.styleSheetLoader._setReferrerPolicy(referrerPolicy);
     }
 
+    const contentCssCors = Options.hasContentCssCors(self);
+    if (Type.isNonNullable(contentCssCors)) {
+      DOMUtils.DOM.styleSheetLoader._setContentCssCors(contentCssCors);
```

Comment:
So I guess the only reason we are comfortable doing this is we think there is no valid reason for someone to have two different css_cors settings on the same page? I can think of a bizarre reason someone might, but is that a limitation we are OK with? Because with this change, interleaving editor loads could have the wrong behaviour if one of them is supposed to have css_cors support, and the other isn't (if I'm understanding the concept properly).

I imagine it's fine ... but it's just something we have to be aware of. We clearly already do this for referrer policy, but you *could* make a case that makes even less sense to not share across all editors. Once again, though, you could find a bizarre edge case where you don't want it to be shared.

Reply:
The other approach was making the editor instantiate its own stylesheetLoader with the editor's own settings/options, but I assume we decided against that?

Reply:
The `content_css_cors` option will now be updated whenever an editor is loaded, and affect any style sheets loaded after it is set, until the option is set to anything different when loading another editor. Inline mode editors share this `styleSheetLoader`, and the document and any stylesheets loaded on it. Now, the stylesheets needed for any editor will be loaded with the correct cors setting, unless it is an inline editor with a different cors setting to a previously initialised inline editor, in which case the stylesheets used by all inline editors will already be loaded.

Reply:
Classic editors have their own `styleSheetLoaders` for stylesheets loaded into the iframe document, however stylesheets loaded onto the page document were not affected by the cors setting.

Reply:
I understand that part. I worry that there is a situation where the stylesheet loading can interleave, and due to the sequencing of promises, the global might be changed before the next stream of stylesheets loads on the first editor. But it's probably unfounded. And as I said before, an edge case. It would depend on the exact ordering of stylesheet loading, and waiting in the editor, to whether there are race conditions here.

Having said that, even if there are race conditions, they would only matter if the editors had different settings, which isn't likely to be the case. I just wanted to know if we were accepting this as a limitation, or we are just confident that there should be no reason due to waits / promises / when stylesheets load that the setting could change before all the stylesheets have loaded for a particular editor.

And my other question was more about did we consider making inline *not* share the global loader, or was that just going to be too big a change?

Reply:
Yeah, that makes some sense, given that the stylesheet will be inherited by each editor regardless of whether it's specified as part of the editor configuration.